app/models/ontology_version/proving.rb

```diff
@@ -0,0 +1,38 @@
+module OntologyVersion::Proving
+
+  extend ActiveSupport::Concern
+  include Hets::ErrorHandling
+
+  included do
+    @queue = 'hets'
+  end
+
+  def async_prove(*args)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `async_prove(*)` if you want the method to accept any arguments but don't care about them.